card/mod/05_standard/set/all/rich_html/editing.rb

```diff
@@ -12,14 +12,8 @@
     end
   end
 
-  def default_new_args args
-    hidden = args[:hidden] ||= {}
-    hidden[:success] ||= card.rule(:thanks) || '_self'
-    hidden[:card] ||= {}
-
-    args[:optional_help] ||= :show
-
-    # name field / title
+  def default_new_args_for_name_field_or_title args
```

Comment:
Assignment Branch Condition size for default_new_args_for_name_field_or_title is too high. [25.98/15]